### ISSUE TITLE
Switching to a dynamic, information-rich homepage

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#2061]
We should consider switching to a dynamic, information-rich homepage to replace the big grey image. 

The goal would be to actively present useful/interesting information to our users, rather than making them dig around.

We could highlight:

- New data, software, models, etc
- Latest news and announcements
- Calls for action (e.g. help us to annotate x, etc).
- Publications that make use of PhysioNet content
- Perhaps leaderboards on benchmark datasets (once benchmarks are added)
- ...

**Comment by mscanlan-git** [comment]
Definitely an interesting project that I've been looking into. I'm not sure what layout we're looking for, but the image should be shrunk as it takes up a lot more screen real estate. Shrinking the image by 50% makes everything look a lot better:

<img width="1512" alt="Screenshot 2023-09-21 at 3 19 34 PM" src="https://github.com/MIT-LCP/physionet-build/assets/67070862/f1fde759-db15-406d-8e70-2fac6b913bec">

(I also removed the down arrow we had, no need for it if things are going to be visible on the same page)

Not sure what we want to do with the columns. If we want to leave them as-is (maybe set a height cap so they're equal size then add things below them), change them to rows, etc. Lots of potential options that can be done.

**Comment by briangow** [comment]
Thanks @mscanlan-git ! Just a couple random thoughts. I don't like the break between the part with the dark image as a background and the columns below. This isn't as much of a problem on the live site because the dark image takes up the entire screen. I think everything should have the dark image as a background or we should remove the dark image entirely. The latter is probably a better choice. In this case, to keep in theme with the content below, perhaps you could use the same light grey outline for the top section.

**Comment by tompollard** [comment]
Yep, thanks for thinking about this Michael! My preference would also be to remove the image altogether. A good starting point might be to look around at other websites for inspiration. A few starters:

- GitHub: https://github.com/
- Reddit: https://www.reddit.com/
- HuggingFace: https://huggingface.co/
- Figshare: https://figshare.com/
- Dryad: https://datadryad.org/

It's interesting that the landing pages for these sites split broadly into two different approaches: 

1. GitHub, Reddit, HuggingFace: fairly busy, high-information, customised to the user.
2. FigShare and Dryad: simple, large image, no customisation to the user (similar to PhysioNet).

As a user, I prefer type 1.

**Comment by mscanlan-git** [comment]
Thanks for your feedback @briangow and @tompollard ! I think if we were to use inspiration from those websites, going with type 1 would be best considering the type of content PhysioNet publishes and the audience interested in said content. I will mess around with some potential layouts and see what makes the most sense and update when I can.
